translate/routers/py_router.py

```diff
@@ -1,3 +1,14 @@
+import sys
+import ast
+sys.path.insert(1,'helpers')
+import jack_py_helpers as h
```

Comment:
Use more descriptive names than h and e for the imports to enhance readability at the call sites.